code/modules/clothing/under/syndicate.dm

```diff
@@ -14,6 +14,7 @@
 	icon_state = "tactifool"
 	item_state = "bl_suit"
 	item_color = "tactifool"
+	has_sensor = HAS_SENSORS //NSV13 - MAA suit sensors

```

Comment:
This is a syndicate item and is only being used by the MAA as a place holder, please do not change the suit sensors on it.

Reply:
This is a subtype of the turtlenecks worn by syndicate operatives, and can be purchased in ClothesMate vendors. I'll remove it for now

Reply:
See below.